tests/phpunit/Helpers/WC_Subscriptions_Product.php

```
@@ -23,12 +23,30 @@ class WC_Subscriptions_Product {
 	 */
 	public static bool $is_subscription_result;
 
+	/**
+	 * When non-empty, is_subscription() returns true only for these product IDs. Used for mixed cart (e.g. multiple simple + one subscription).
+	 *
+	 * @var int[]
+	 */
+	public static array $subscription_product_ids = [];
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $subscription_product_ids. Keep variable name length under 20. (undefined)

(LongVariable)